src/openrct2-ui/interface/FileBrowser.cpp

```diff
@@ -24,6 +24,7 @@
 #include <openrct2/platform/Platform.h>
 #include <openrct2/rct2/T6Exporter.h>
 #include <openrct2/ride/TrackDesign.h>
+#include <openrct2/scenario/Scenario.h>
```

Comment:
> Move Scenario::Objective into its own unit; enum class ObjectiveType

Shouldn't a lot of places that are using `Scenario::Objective` have gained its new include then? (and potentially removed the `Scenario.h`, in case it was only there for this reason)

I would expect all places using the new types to gain the includes.

Reply:
No, it's the other way around. This unit actually needs functions from `Scenario.h`, e.g. to load saves. As mentioned in the OP, this include previously happened implicitly due to `GameState.h` including it.